### PR TITLE
New cal csvs

### PR DESCRIPTION
**Pull request by rtravis-whoi**

### CODE REVIEW

File: calibration/OPTAAD/CGINS-OPTAAD-00193__20240613__CC_taarray.ext [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+-0.005743,-0.004308,-0.002900,-0.002811,-0.002557,-0.002267,-0.002328,-0.002189,-0.002553,-0.002344,-0.001892,-0.001858,-0.001772,-0.001570,-0.001469,-0.000964,-0.001055,-0.001075,-0.000645,-0.000652,-0.000666,-0.000468,-0.000372,-0.000398,0.000142,0.000000,-0.000037,-0.000144,0.000090,0.000005,0.000127,-0.000101,-0.000081,0.000084,-0.000342
+-0.005794,-0.004395,-0.003041,-0.002871,-0.002722,-0.002511,-0.002695,-0.002329,-0.002734,-0.002539,-0.002183,-0.001931,-0.002020,-0.001793,-0.001649,-0.001178,-0.001145,-0.001251,-0.000678,-0.000774,-0.000766,-0.000592,-0.000423,-0.000422,0.000052,0.000000,-0.000159,-0.000190,0.000026,0.000007,0.000024,-0.000091,-0.000104,0.000056,-0.000351
+-0.006047,-0.004641,-0.003209,-0.003165,-0.002936,-0.002622,-0.002702,-0.002571,-0.002977,-0.002791,-0.002150,-0.002199,-0.002040,-0.001976,-0.001884,-0.001279,-0.001460,-0.001321,-0.000742,-0.000874,-0.000912,-0.000649,-0.000636,-0.000575,-0.000017,0.000000,-0.000146,-0.000247,-0.000087,0.000020,0.000047,-0.000100,-0.000086,0.000042,-0.000405
```
Comment (cooleyky) [inline calibration/OPTAAD/CGINS-OPTAAD-00193__20240613__CC_taarray.ext:88]: spot checked, looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00193__20240613__CC_tcarray.ext [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+0.016887,0.013965,0.012310,0.011650,0.011176,0.010162,0.009059,0.008034,0.007879,0.007364,0.006874,0.006412,0.005824,0.005307,0.004651,0.004037,0.003582,0.003025,0.002835,0.002435,0.001862,0.001501,0.001033,0.000690,0.000301,0.000000,-0.000131,-0.000457,-0.000614,-0.000942,-0.001165,-0.001354,-0.001436,-0.001800,-0.001766
+0.016431,0.013766,0.012061,0.011122,0.010683,0.009527,0.008351,0.007530,0.007134,0.006679,0.006123,0.005662,0.005272,0.004603,0.004215,0.003850,0.003294,0.002890,0.002525,0.002159,0.001610,0.001220,0.000882,0.000526,0.000283,0.000000,-0.000458,-0.000446,-0.000918,-0.001288,-0.001440,-0.001457,-0.001840,-0.002101,-0.002082
+0.017170,0.014155,0.012881,0.011907,0.011391,0.010453,0.009064,0.008103,0.007845,0.007216,0.006725,0.006047,0.005595,0.004965,0.004448,0.004138,0.003436,0.002898,0.002705,0.002296,0.001575,0.001400,0.001042,0.000500,0.000259,0.000000,-0.000219,-0.000522,-0.000608,-0.001055,-0.001193,-0.001542,-0.001638,-0.001895,-0.001799
```
Comment (cooleyky) [inline calibration/OPTAAD/CGINS-OPTAAD-00193__20240613__CC_tcarray.ext:88]: spot checked looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00209__20240702.csv [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+ACS-209,CC_tbins,"[0.529950, 1.500326, 2.449216, 3.454444, 4.496190, 5.486875, 6.520952, 7.498400, 8.483478, 9.495000, 10.475217, 11.508800, 12.497391, 13.500000, 14.493333, 15.482917, 16.498333, 17.502800, 18.506667, 19.490400, 20.491154, 21.500800, 22.503750, 23.505652, 24.486538, 25.495200, 26.502000, 27.494400, 28.512308, 29.513600, 30.493333, 31.471250, 32.467895, 33.440625, 34.484000]",
+ACS-209,CC_taarray,SheetRef:CC_taarray,
+ACS-209,CC_tcarray,SheetRef:CC_tcarray,
```
Comment (cooleyky) [inline calibration/OPTAAD/CGINS-OPTAAD-00209__20240702.csv:9]: spot checked, looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00209__20240702__CC_taarray.ext [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+-0.005771,-0.004660,-0.003990,-0.003897,-0.003811,-0.003773,-0.003660,-0.003423,-0.003437,-0.003117,-0.002933,-0.002598,-0.002405,-0.002201,-0.002189,-0.001994,-0.001945,-0.001763,-0.001857,-0.001557,-0.001324,-0.000895,-0.000768,-0.000341,-0.000136,0.000000,0.000086,0.000230,0.000430,0.000228,0.000562,0.000707,0.000721,0.000928,0.000870
+-0.005873,-0.004787,-0.004067,-0.004036,-0.003926,-0.003704,-0.003917,-0.003614,-0.003588,-0.003233,-0.003031,-0.002691,-0.002608,-0.002403,-0.002324,-0.002189,-0.002047,-0.001965,-0.001890,-0.001770,-0.001516,-0.001185,-0.000931,-0.000461,-0.000141,0.000000,0.000070,0.000170,0.000330,0.000314,0.000693,0.000604,0.000626,0.000800,0.000787
+-0.006217,-0.005074,-0.004371,-0.004213,-0.004123,-0.003966,-0.003977,-0.003753,-0.003830,-0.003393,-0.003194,-0.002848,-0.002655,-0.002511,-0.002420,-0.002212,-0.002037,-0.002029,-0.001950,-0.001736,-0.001499,-0.001184,-0.000846,-0.000418,-0.000181,0.000000,0.000090,0.000224,0.000326,0.000210,0.000533,0.000684,0.000562,0.000953,0.000783
```
Comment (cooleyky) [inline calibration/OPTAAD/CGINS-OPTAAD-00209__20240702__CC_taarray.ext:86]: spot checked, looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00209__20240702__CC_tcarray.ext [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+0.003941,0.003728,0.003131,0.003037,0.002754,0.002624,0.002274,0.002310,0.002170,0.002209,0.002042,0.001994,0.001667,0.001516,0.001309,0.001128,0.001054,0.000704,0.000591,0.000422,0.000369,0.000397,0.000137,0.000049,0.000082,0.000000,-0.000254,-0.000344,-0.000691,-0.000859,-0.001183,-0.001545,-0.001714,-0.001993,-0.002095
+0.006586,0.006531,0.005418,0.004442,0.003979,0.003513,0.003540,0.003589,0.003248,0.002808,0.002656,0.002508,0.002336,0.002328,0.002041,0.001823,0.001771,0.001615,0.001492,0.001319,0.001033,0.000581,0.000596,0.000167,0.000135,0.000000,-0.000265,-0.000670,-0.000692,-0.000812,-0.000979,-0.001183,-0.001173,-0.001285,-0.000940
+0.007518,0.007414,0.006527,0.005647,0.004895,0.004659,0.004450,0.004379,0.004025,0.003721,0.003606,0.003330,0.003097,0.003069,0.002547,0.002348,0.002087,0.001545,0.001469,0.001318,0.001067,0.000784,0.000534,0.000327,0.000110,0.000000,-0.000248,-0.000416,-0.000847,-0.000601,-0.000891,-0.000837,-0.000663,-0.000421,0.000154
```
Comment (cooleyky) [inline calibration/OPTAAD/CGINS-OPTAAD-00209__20240702__CC_tcarray.ext:86]: spot checked, looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00254__20240701.csv [hunk 6]
```diff
@@ -0,0 +1,9 @@
...
+ACS-254,CC_tbins,"[0.754056, 1.300683, 2.446809, 3.456786, 4.486364, 5.534737, 6.478750, 7.510417, 8.487826, 9.492917, 10.473333, 11.500417, 12.501739, 13.485417, 14.492500, 15.496957, 16.473750, 17.493200, 18.488750, 19.489231, 20.490741, 21.485417, 22.478696, 23.483600, 24.482400, 25.494400, 26.493462, 27.495000, 28.481429, 29.501923, 30.499167, 31.488148, 32.489643, 33.484444, 34.515000, 35.000000]",
+ACS-254,CC_taarray,SheetRef:CC_taarray,
+ACS-254,CC_tcarray,SheetRef:CC_tcarray,
```
Comment (cooleyky) [inline calibration/OPTAAD/CGINS-OPTAAD-00254__20240701.csv:9]: spot checked, looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00254__20240701__CC_taarray.ext [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+-0.005525,-0.004073,-0.002948,-0.002360,-0.002343,-0.002204,-0.001919,-0.002554,-0.002350,-0.002057,-0.001733,-0.001655,-0.001459,-0.001378,-0.001294,-0.001035,-0.000960,-0.000802,-0.000614,-0.000757,-0.000589,-0.000340,-0.000219,-0.000218,-0.000294,0.000000,0.000045,-0.000083,-0.000028,-0.000137,-0.000181,-0.000379,-0.000569,-0.000782,-0.000828,-0.001191
+-0.005780,-0.004326,-0.003267,-0.002680,-0.002727,-0.002462,-0.002168,-0.002840,-0.002430,-0.002251,-0.001986,-0.001926,-0.001404,-0.001543,-0.001349,-0.000968,-0.001064,-0.000870,-0.000618,-0.000790,-0.000592,-0.000371,-0.000235,-0.000305,-0.000324,0.000000,0.000004,-0.000019,-0.000036,-0.000184,-0.000066,-0.000336,-0.000458,-0.000694,-0.000741,-0.000955
+-0.005763,-0.004178,-0.002913,-0.002283,-0.002324,-0.002249,-0.002117,-0.002708,-0.002481,-0.002213,-0.001941,-0.001827,-0.001444,-0.001555,-0.001438,-0.001086,-0.000904,-0.000684,-0.000698,-0.000851,-0.000697,-0.000276,-0.000204,-0.000130,-0.000256,0.000000,0.000107,-0.000090,0.000151,-0.000088,-0.000082,-0.000237,-0.000378,-0.000697,-0.000771,-0.000878
```
Comment (cooleyky) [inline calibration/OPTAAD/CGINS-OPTAAD-00254__20240701__CC_taarray.ext:87]: spot checked, looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00254__20240701__CC_tcarray.ext [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
+0.001113,0.000716,0.001263,0.001274,0.001537,0.001901,0.001654,0.001584,0.001343,0.001467,0.001418,0.001380,0.001314,0.001096,0.001053,0.001127,0.000672,0.000654,0.000657,0.000645,0.000514,0.000406,0.000362,0.000203,0.000103,0.000000,-0.000079,-0.000024,-0.000028,-0.000343,-0.000670,-0.000866,-0.001282,-0.001625,-0.002099,-0.001816
+0.003506,0.002914,0.003301,0.003064,0.003099,0.003021,0.001984,0.001776,0.001862,0.001619,0.001422,0.001235,0.001317,0.001257,0.001260,0.000835,0.000875,0.000764,0.000564,0.000440,0.000390,0.000414,0.000316,0.000125,0.000061,0.000000,-0.000147,-0.000092,-0.000146,-0.000154,-0.000341,-0.000356,-0.000474,-0.001009,-0.001416,-0.001486
+0.004735,0.003679,0.003578,0.003348,0.003032,0.002857,0.002538,0.002035,0.001925,0.001645,0.001575,0.001366,0.001127,0.001041,0.000948,0.000909,0.000652,0.000588,0.000366,0.000391,0.000299,0.000268,0.000222,0.000045,0.000040,0.000000,0.000005,0.000015,-0.000054,-0.000013,-0.000113,-0.000237,-0.000194,-0.000272,-0.000640,-0.000844
```
Comment (cooleyky) [inline calibration/OPTAAD/CGINS-OPTAAD-00254__20240701__CC_tcarray.ext:87]: spot checked, looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00255__20240710.csv [hunk 9]
```diff
@@ -0,0 +1,9 @@
...
+ACS-255,CC_tbins,"[0.863860, 1.407570, 2.431228, 3.437931, 4.455000, 5.472857, 6.457222, 7.537500, 8.499583, 9.521667, 10.510000, 11.493750, 12.499167, 13.504800, 14.501304, 15.490000, 16.493333, 17.508261, 18.501200, 19.487083, 20.465600, 21.472800, 22.498846, 23.507083, 24.492083, 25.491154, 26.504167, 27.500800, 28.488462, 29.505600, 30.494000, 31.470000, 32.467273, 33.480588, 34.500000]",
+ACS-255,CC_taarray,SheetRef:CC_taarray,
+ACS-255,CC_tcarray,SheetRef:CC_tcarray,
```
Comment (cooleyky) [inline calibration/OPTAAD/CGINS-OPTAAD-00255__20240710.csv:9]: spot checked, looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00255__20240710__CC_taarray.ext [hunk 10]
```diff
@@ -0,0 +1,88 @@
...
+0.002117,0.001897,0.002028,0.001976,0.001830,0.001759,0.001762,0.001910,0.001700,0.001470,0.001313,0.001244,0.000973,0.001162,0.001063,0.000892,0.000701,0.000494,0.000497,0.000607,0.000222,0.000443,0.000285,0.000279,0.000117,0.000000,0.000092,-0.000035,-0.000172,-0.000224,-0.000219,-0.000221,-0.000387,-0.000546,-0.000551
+0.002017,0.001954,0.001917,0.001958,0.001712,0.001586,0.001610,0.001697,0.001611,0.001420,0.001216,0.001044,0.000931,0.001061,0.000933,0.000874,0.000687,0.000458,0.000470,0.000630,0.000347,0.000358,0.000195,0.000375,-0.000041,0.000000,0.000074,-0.000060,-0.000070,-0.000123,-0.000086,-0.000052,-0.000208,-0.000396,-0.000308
+0.001885,0.001628,0.001805,0.001739,0.001524,0.001383,0.001466,0.001581,0.001384,0.001151,0.001087,0.000863,0.000771,0.000881,0.000784,0.000503,0.000387,0.000259,0.000176,0.000331,0.000057,0.000189,0.000034,0.000078,-0.000172,0.000000,-0.000078,-0.000243,-0.000351,-0.000336,-0.000205,-0.000364,-0.000533,-0.000645,-0.000529
```
Comment (cooleyky) [inline calibration/OPTAAD/CGINS-OPTAAD-00255__20240710__CC_taarray.ext:88]: spot checked, looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00255__20240710__CC_tcarray.ext [hunk 11]
```diff
@@ -0,0 +1,88 @@
...
+0.008198,0.007999,0.007797,0.007272,0.006954,0.006519,0.006227,0.005891,0.005592,0.005245,0.005043,0.004802,0.004617,0.004232,0.003646,0.003283,0.002966,0.002554,0.002255,0.001902,0.001671,0.001480,0.000969,0.001021,0.000520,0.000000,-0.000355,-0.000757,-0.000889,-0.001179,-0.001496,-0.001628,-0.001741,-0.001865,-0.001781
+0.006772,0.007045,0.006859,0.006950,0.006635,0.006700,0.006355,0.005910,0.005495,0.005374,0.005241,0.004662,0.004155,0.003881,0.003793,0.003438,0.003023,0.002762,0.002571,0.001965,0.001390,0.001211,0.000793,0.000326,0.000306,0.000000,-0.000337,-0.000617,-0.000869,-0.001299,-0.001415,-0.001808,-0.001813,-0.001832,-0.001739
+0.007320,0.007458,0.006464,0.006152,0.005768,0.005421,0.005481,0.005335,0.005002,0.004698,0.004444,0.004175,0.004173,0.003591,0.003166,0.002758,0.002658,0.002296,0.001960,0.001831,0.001631,0.001183,0.000916,0.000605,0.000261,0.000000,-0.000434,-0.000588,-0.000783,-0.000869,-0.001297,-0.001495,-0.001390,-0.001630,-0.001674
```
Comment (cooleyky) [inline calibration/OPTAAD/CGINS-OPTAAD-00255__20240710__CC_tcarray.ext:88]: spot checked, looks good

**Review by cooleyky — COMMENTED** [review]
OPTAA cal csvs look good

File: calibration/METBKA/CGINS-METCTD-13546__20240529.csv [hunk 12]
```diff
@@ -0,0 +1,12 @@
...
+37-13546,CC_a1,2.971690e-004,
+37-13546,CC_a2,-3.719458e-006,
+37-13546,CC_a3,1.802135e-007,
+37-13546,CC_g,-1.010525e+000,
+37-13546,CC_h,1.388909e-001,
+37-13546,CC_i,-3.184025e-005,
+37-13546,CC_j,2.455662e-005,
+37-13546,CC_ctcor,3.250000e-006,
+37-13546,CC_cpcor,-9.570000e-008,
+37-13546,CC_wbotc,-9.270967e-008,
```
Comment (cooleyky) [inline calibration/METBKA/CGINS-METCTD-13546__20240529.csv:12]: looks good

File: calibration/PCO2WB/CGINS-PCO2WB-C0070__20240614.csv [hunk 13]
```diff
@@ -0,0 +1,11 @@
...
+C0070,CC_cala,0.0624,source_file PCO2W-B_SAMI-CO2_SN_C0070_Calibration_Files_2024-06-14.zip > c70_06_CalCert.pdf
+C0070,CC_calb,0.5859,
+C0070,CC_calc,-1.5850,
+C0070,CC_calt,10.3220,
+C0070,CC_ea434,19706,constant
+C0070,CC_ea620,34,constant
+C0070,CC_eb434,3073,constant
+C0070,CC_eb620,44327,constant
+C0070,CC_sami_bits,12,
+C0070,CC_cal_range,"[204,986]",
```
Comment (cooleyky) [inline calibration/PCO2WB/CGINS-PCO2WB-C0070__20240614.csv:11]: looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00254__20240701.csv (hunk 6 above)
Comment (cdobs) [inline calibration/OPTAAD/CGINS-OPTAAD-00254__20240701.csv:9]: looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00254__20240701__CC_taarray.ext (hunk 7 above)
Comment (cdobs) [inline calibration/OPTAAD/CGINS-OPTAAD-00254__20240701__CC_taarray.ext:87]: spot checked, looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00254__20240701__CC_tcarray.ext (hunk 8 above)
Comment (cdobs) [inline calibration/OPTAAD/CGINS-OPTAAD-00254__20240701__CC_tcarray.ext:87]: spot checked, looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00255__20240710.csv (hunk 9 above)
Comment (cdobs) [inline calibration/OPTAAD/CGINS-OPTAAD-00255__20240710.csv:9]: spot checked, looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00255__20240710__CC_taarray.ext (hunk 10 above)
Comment (cdobs) [inline calibration/OPTAAD/CGINS-OPTAAD-00255__20240710__CC_taarray.ext:88]: spot checked, looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00255__20240710__CC_tcarray.ext (hunk 11 above)
Comment (cdobs) [inline calibration/OPTAAD/CGINS-OPTAAD-00255__20240710__CC_tcarray.ext:88]: spot checked, looks good

File: calibration/PCO2WB/CGINS-PCO2WB-C0070__20240614.csv (hunk 13 above)
Comment (cdobs) [inline calibration/PCO2WB/CGINS-PCO2WB-C0070__20240614.csv:11]: looks good

File: calibration/PCO2WB/CGINS-PCO2WB-C0071__20240614.csv [hunk 14]
```diff
@@ -0,0 +1,11 @@
...
+C0071,CC_cala,0.0350, source_file PCO2W-B_SAMI-CO2_SN_C0071_Calibration_Files_2024-06-14.zip > c71_06_CalCert.pdf
+C0071,CC_calb,0.6784,
+C0071,CC_calc,-1.6568,
+C0071,CC_calt,10.1226,
+C0071,CC_ea434,19706,constant
+C0071,CC_ea620,34,constant
+C0071,CC_eb434,3073,constant
+C0071,CC_eb620,44327,constant
+C0071,CC_sami_bits,12,
+C0071,CC_cal_range,"[204,986]",
```
Comment (cdobs) [inline calibration/PCO2WB/CGINS-PCO2WB-C0071__20240614.csv:11]: looks good

File: calibration/PCO2WB/CGINS-PCO2WB-C0127__20240614.csv [hunk 15]
```diff
@@ -0,0 +1,11 @@
...
+C0127,CC_cala,0.0363,source_file PCO2W-B_SAMI-CO2_SN_C0127_Calibration_Files_2024-06-14.zip > c127_06_CalCert.pdf
+C0127,CC_calb,0.6683,
+C0127,CC_calc,-1.6364,
+C0127,CC_calt,10.3105,
+C0127,CC_ea434,19706,constant
+C0127,CC_ea620,34,constant
+C0127,CC_eb434,3073,constant
+C0127,CC_eb620,44327,constant
+C0127,CC_sami_bits,12,
+C0127,CC_cal_range,"[204,987]",
```
Comment (cdobs) [inline calibration/PCO2WB/CGINS-PCO2WB-C0127__20240614.csv:11]: looks good

File: calibration/PCO2WB/CGINS-PCO2WB-C0135__20240614.csv [hunk 16]
```diff
@@ -0,0 +1,11 @@
...
+C0135,CC_cala,0.0201,source_file PCO2W-B_SAMI-CO2_SN_C0135_Calibration_Files_2024-06-14.zip > c135_06_CalCert.pdf
+C0135,CC_calb,0.7590,
+C0135,CC_calc,-1.7629,
+C0135,CC_calt,10.3428,
+C0135,CC_ea434,19706,constant
+C0135,CC_ea620,34,constant
+C0135,CC_eb434,3073,constant
+C0135,CC_eb620,44327,constant
+C0135,CC_sami_bits,12,
+C0135,CC_cal_range,"[204,986]",
```
Comment (cdobs) [inline calibration/PCO2WB/CGINS-PCO2WB-C0135__20240614.csv:11]: looks good

**Review by cdobs — APPROVED** [review]

**Pull request merged**